### DISCUSSION TITLE
not getting amp settings...

### DISCUSSION THREAD

**Discussion by wader2k** [soundshed/soundshed-app#62] [General]
Trying to get this to work with new Spark 40. Windows 10 64bit - paired Spark BLE and Spark Audio via bluetooth - no problem. 
Web version of soundshed sees amp but i don't see any of the Amp Controls. The 4 presets show up and ARE active. Selecting the presets changes the Spark settings. No Amp Control shows up below the preset section however. I downloaded the software to eliminate web version issues and I can't get the app version  to see the amp at all. Really wanted to get this working as I HATE cellphones and tablets. Is there anything I can try to get this working?!

**Comment by wader2k** [comment]
btw  This is the message I get below the Preset buttons area:  "No preset selected (or amp not connected). Connect and refresh to get current amp settings. You may need to select a preset button on the amp to start.null" 
I select preset 1 on the amp and in soundshed and refresh but it never gets beyond this point on the Web version(using Edge browser). The preset buttons ARE active and change the presets on amp.

**Reply by webprofusion-chrisc** [reply]
Hi, there is an occasional (but quite frequent) glitch while reading the preset from the amp, I'm actually working on improving that stuff just now.

For now I would try hitting the Refresh button in the app and if that doesn't work reload the page and reconnect. If the problem is consistent try switching your amp off and on and/or forgetting the device in Windows and re-pairing it.

**Reply by wader2k** [reply]
Got it to work this morning in the Browser version!!  Very pleased. I was beginning to think my BT(built into Win 10) wasn't a high enough version. Now I'm afraid to turn it off<grin>.  One question ... once i have a sound i like how do i save it in the app or is the only way to actually use the preset button on the amp to save?
